rubocop.gemspec

```diff
@@ -39,7 +39,7 @@ Gem::Specification.new do |s|
   s.summary = 'Automatic Ruby code style checking tool.'
 
   s.add_runtime_dependency('rainbow', '>= 1.99.1', '< 3.0')
-  s.add_runtime_dependency('parser', '>= 2.2.0.1', '< 3.0')
+  s.add_runtime_dependency('parser', '>= 2.2.2.1', '< 3.0')
```

Comment:
@bbatsov showed me that there's another fix we want as well: https://github.com/whitequark/parser/commit/a5e35eceec4c4006b29475e764600838f7b3ac46

I forgot to update the changelog. 2.2.0.4 is the version that fixes the problem at hand, though, so I think I'll just make the log entry more vague.